cassandra-analytics-core/build.gradle

```diff
@@ -77,8 +77,7 @@ dependencies {
     runtimeOnly(group: 'net.java.dev.jna', name: 'jna', version: "${jnaVersion}")
     runtimeOnly(group: 'net.java.dev.jna', name: 'jna-platform', version: "${jnaVersion}")
 
-    // This dependency must be built by running `scripts/build-dependencies.sh`
-    api(group: "${sidecarClientGroup}", name: "${sidecarClientName}", version: "${sidecarVersion}")
```

Comment:
there's clean up to be  done for these variables. I see it in the profiles + settings.gradle. It'd be useful to clean that up as part of this PR, since we are removing usage of them here